classes/check/tasklatencycheck.php

```diff
@@ -34,7 +34,6 @@
 use core\check\result;
 
 defined('MOODLE_INTERNAL') || die();
-require_once("$CFG->libdir/cronlib.php");
```

Comment:
Since this plugin exists only with 1 branch which supports all the way back to 2.7, removing this would break the plugin in older versions.

The best way to mitigate this is to wrap this in a conditional check based on the version number